From 90b18b3f7694e6ef53387913905e5a357d43ffb6 Mon Sep 17 00:00:00 2001
From: Zdenek Kabelac <zkabelac@redhat.com>
Date: Sat, 25 Oct 2025 22:20:22 +0200
Subject: [PATCH 12/32] man: keep page ascii

Drop utf8 characters.  Can be captured with:

$ grep -r -P '[^\x00-\x7f]'

Converted with:

$ iconv -f utf-8 -t ascii//TRANSLIT file > file_ascii

(cherry picked from commit 7a18fc86cedd9b27d636fa9f5857966460f6465a)
---
 man/lvmpersist.8_main | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/lvmpersist.8_main b/man/lvmpersist.8_main
index e05f25a9e..785e0ed97 100644
--- a/man/lvmpersist.8_main
+++ b/man/lvmpersist.8_main
@@ -71,16 +71,16 @@ Write Exclusive
 Exclusive Access
 .br
 .B WERO
-Write Exclusive – registrants only
+Write Exclusive - registrants only
 .br
 .B EARO
-Exclusive Access – registrants only
+Exclusive Access - registrants only
 .br
 .B WEAR
-Write Exclusive – all registrants
+Write Exclusive - all registrants
 .br
 .B EAAR
-Exclusive Access – all registrants
+Exclusive Access - all registrants
 .P
 (WERO and EARO types are not yet supported.)
 .P
-- 
2.52.0

